telluric/georaster.py

```diff
@@ -37,8 +41,10 @@
 from telluric.vectors import GeoVector
 from telluric.util.projections import transform
 
-from telluric.util.raster_utils import (convert_to_cog, _calc_overviews_factors,
-                                        _mask_from_masked_array, _join_masks_from_masked_array)
+from telluric.util.raster_utils import (
+    convert_to_cog, _calc_overviews_factors,
+    _mask_from_masked_array, _join_masks_from_masked_array,
```

Comment:
I wonder whether these functions should be private. Are they used only here, or in more places?

Reply:
What function do you mean?

Reply:
`_calc_overviews_factors`, `_mask_from_masked_array` and `_join_masks_from_masked_array` in this case. If they are private they should not be imported from the outside I think, but I didn't go a carefully read everything so perhaps it's worth it to make an exception here. It doesn't block this PR anyway.

Reply:
I'm not using these functions in my code.